androidClient/app/src/main/java/com/csatimes/dojma/activities/VideosActivity.java

```diff
@@ -0,0 +1,60 @@
+package com.csatimes.dojma.activities;
+
+
+
+import androidx.appcompat.app.AppCompatActivity;
+import android.os.Bundle;
+import android.widget.TextView;
+
+import com.csatimes.dojma.R;
+import com.pierfrancescosoffritti.youtubeplayer.player.AbstractYouTubePlayerListener;
+import com.pierfrancescosoffritti.youtubeplayer.player.YouTubePlayer;
+import com.pierfrancescosoffritti.youtubeplayer.player.YouTubePlayerInitListener;
+import com.pierfrancescosoffritti.youtubeplayer.player.YouTubePlayerView;
+import com.pierfrancescosoffritti.youtubeplayer.player.*;
+
+public class VideosActivity extends AppCompatActivity {
+    private String videoID;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_videos);
+        //getActionBar().hide();
+        videoID= getIntent().getStringExtra("id");
+        String dateStamp = getIntent().getStringExtra("date");
```

Comment:
create a `public static` method in **`VideosActivity`** class to launch the activity with required parameters. (date, title, etc)